### PR TITLE
chore: reorganize ZAC Flowable backend code

### PR DESCRIPTION
**Pull request by edgarvonk**
Reorganized ZAC Flowable backend code a little. Moved around classes to more logical packages. Added unit test.

Solves PZ-3634

### CODE REVIEW

File: src/main/java/net/atos/zac/flowable/cmmn/CMMNService.java [hunk 1]
```diff
@@ -1,23 +1,23 @@
 /*
- * SPDX-FileCopyrightText: 2022 Atos
+ * SPDX-FileCopyrightText: 2024 Lifely
```
Comment (infonl-marcel) [inline src/main/java/net/atos/zac/flowable/cmmn/CMMNService.java:2]: Should this be `SPDX-FileCopyrightText: 2022 Atos, 2024 Lifely` ?

**Review by infonl-marcel — APPROVED** [review]

File: src/main/java/net/atos/zac/flowable/cmmn/CMMNService.java (hunk 1 above)
Comment (edgarvonk) [reply src/main/java/net/atos/zac/flowable/cmmn/CMMNService.java:2]: oei!

Comment (infonl-marcel) [reply src/main/java/net/atos/zac/flowable/cmmn/CMMNService.java:2]: 👍🏽 I noticed more cases in this PR BTW; you want me to make edits?

Comment (edgarvonk) [reply src/main/java/net/atos/zac/flowable/cmmn/CMMNService.java]: fix them now hopefully!

**Pull request merged**